### ISSUE TITLE
Request: Support for other email services (Sendgrid, Mailgun, etc) 

### ISSUE THREAD

**Issue by hdpinto** [keystonejs/keystone#2651]
Mandrill is now paid-only, requires a paid mailchimp account plus a mandrill add-on that starts at US$20/month. Basically, I think it would be best to allow developers to choose a mail system rather than baking one into KeystoneJS.

**Comment by sashimii** [comment]
I'd like to second this. More than one option for emails would be ideal.  Mailgun is one I've used in the past. Nodemailer works fine too, giving the developer the option to set up their own SMTP configuration

**Comment by mxstbr** [comment]
Duplicate of #250.

TL;DR: We're aware and will release a fix as soon as the main 0.4 things are done—if you want to help we'd much appreciate it, I'm currently a bit locked up finishing off #2566.

*Also #2627, #2624, #2623, #2338, #2332*

**Comment by webteckie** [comment]
@mxstbr the only thing I did to get around it is add this dummy line to init:
```'
cloudinary config': 'cloudinary://api_key:api_secret@cloud_name',
```

You can try it with the e2e server:

`node test/e2e/server.js --notest`
